pages/common/mysqldump.md

```diff
@@ -3,18 +3,18 @@
 > Backups MySQL databases.
 > More information: <https://dev.mysql.com/doc/refman/en/mysqldump.html>.
 
-- Create a backup, user will be prompted for a password:
+- Create a backup (user will be prompted for a password):
 
-`mysqldump -u {{user}} -p {{database_name}} -r {{filename.sql}}`
+`mysqldump --user {{user}} --password {{database_name}} -r {{filename.sql}}`
 
-- Restore a backup, user will be prompted for a password:
+- Restore a backup (user will be prompted for a password):
 
-`mysql -u {{user}} -p {{database_name}} < {{filename.sql}}`
+`mysqldump --user {{user}} --password {{database_name}} < {{path/to/file.sql}}`
```

Comment:
I have a doubt about this one, tested it now and it doesn't wok, are you sure that mysqldump (not mysql) allow this syntax to import a dump ?

Reply:
Oh, really? I assumed that was just a typo on the command name :thinking: so you're saying this is actually a `mysql` command?

Reply:
yes this is a mysql command :p

Reply:
Does it actually restore a backup though? Because from what I can see on [the man page](https://linux.die.net/man/1/mysql) it seems like that just reads an SQL script to execute on the given database:

> You can execute SQL statements in a script file (batch file) like this:
>
>     shell> mysql db_name < script.sql > output.tab

Reply:
yes, my original patch restore a dump cf `mysql -u {{user}} -p {{database_name}} < {{filename.sql}}`

Reply:
Thank you, I'll move that to the `mysql` page then since it's a different command.

Reply:
I understand, but it seems a bit pitty since i was proposing to refine an item on mysqldump page, not to remove it :\

People that will look to mysqldump will loose this info that was available on this page.

Reply:
Well that's a different command though. If `mysqldump` can't restore backups I would suggest to change the description of this page to something like:

> Backup MySQL databases to be later restored through the `mysql` command.

What do you think?

Reply:
Or keep an item about dump restoring without any command but instead a mention pointing to mysql page ?

Reply:
I'm afraid that'd go against the format of the pages. On the other hand, some pages have mention of other "companion" pages in their description.

Reply:
On this discussion: We can use the `see also` that is used in multiple pages around tldr (eg. [1](pages/linux/sinfo.md) ). 
Something like `See also mysql for more information on restoring databases`

Reply:
Full example:

```markdown
> See also `mysql` for restoring databases.
```

Reply:
See #3912.